turbo.json

```
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "start": {
+      "cache": false,
+      "persistent": true
```

Needs to depend on build
```suggestion
      "persistent": true,
      "dependsOn": ["^build"]
```